### PR TITLE
updated divya.md file

### PR DESCRIPTION
**Pull request by Divshub13**
TL;DR this pr is updated the copy of md file

## Done
- made a commit
- created a branch

## To do
- [] merge the branch
- [] delete branch later

### CODE REVIEW

File: _slides/02-Divshub13.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
 ---
 
-Caption here
+Hello from Divya
```
Comment (Divshub13) [inline _slides/02-Divshub13.md:6]: Looks good

Comment (Divshub13) [inline _slides/02-Divshub13.md:6]: lets talk about it

Comment (Divshub13) [inline _slides/02-Divshub13.md:6]: Lets talk about it

**Review by Divshub13 — COMMENTED** [review]
ok

**Review by adilshraddha — APPROVED** [review]
Approved pull request

**Pull request merged**